internal/orchestrator/applications_handler_test.go

```diff
@@ -79,6 +87,22 @@ func TestListApps(t *testing.T) {
 	})
 }
 
+func TestListApps_withSort(t *testing.T) {
+	testsupport.WithSetUp(&applicationsHandlerData{}, func(data *applicationsHandlerData) {
+		url := fmt.Sprintf("http://%s/applications", data.server.Addr)
+		resp, _ := hawksupport.HawkGet(&http.Client{}, "anId", data.key, url)
+		assert.Equal(t, http.StatusOK, resp.StatusCode)
+
+		var apps orchestrator.Applications
+		_ = json.NewDecoder(resp.Body).Decode(&apps)
+		assert.Equal(t, 4, len(apps.Applications))
+
+		for _, application := range apps.Applications {
+			fmt.Println(application)
```

Comment:
@jingchu000 - This looks great. One last feedback
Please remove the fmt.Println and assert that the applications are actually in sorted order.
It could be as simple as
```
expApps := [][]string{ 
  {"service1": "app1a"}, {"service1": "app1b"}, {"service2": "app2a"}, {"service2": "app2b"}

for a, actApp := range apps.Applications {
    assert.Equal(t, expApps[a][0], actApp.ServiceName)
    assert.Equal(t, expApps[a][1], actApp.Name)
}
```

Reply:
good, learn a lot。

Reply:
Thanks